_pages/federal-fieldguide/planning.md

```diff
@@ -4,8 +4,561 @@ title: Federal field guide
+
+However, remote collaboration can be difficult for agencies. Agency network restrictions and software approval policies make collaboration with remote team members difficult. Many commonly used tools for video-conferencing are blocked in government buildings. Federal employees often resort to workarounds, like using their personal mobile phones, to get access to the services they need. 
+
+Without a set of commonly available collaboration tools, especially for video-conferencing, it is much more difficult for agency teams and contractors to practice Agile development, where frequent communication and feedback are needed to facilitate quick decision-making and prioritization of tasks. The product owner should always know what the team is working on, and the team should be readily available to answer questions or huddle to problem-solve issues as they arise. Every sprint, there should be a demonstration of functioning software. Working like this via phone calls, emails, and PowerPoint presentations is _incredibly — and needlessly — limiting_ for a distributed Agile team.
+
+In the past, our teams have used the following tools to support remote work. See this [article](https://digital.gov/2017/10/03/3-ways-manage-research-projects-remotely/) to learn more about managing teams remotely. Some of the tools 18F uses:
+
+-   A virtual room like Google Hangouts or Appear.in
+
+-   Mural or a similar sticky-note tool
+
+-   A collaborative writing and editing tool like Google Docs
+
+-   A project planning tool like Trello, Jira, or ZenHub 
+
+-   Code repositories like GitHub, Bucket, etc.
+
+Agencies should determine which collaboration tools their teams need and make those available. As an interim step, agencies may want to develop a provisional Authorization to Operate (ATO) process for piloting tools that are relatively low risk before figuring out which tools should go through the ATO process to be rolled out more broadly. 
+
+### Invest in technology incrementally and budget for risk mitigation prototyping. 
+
+Challenge: Budgeting for large, risky investments in major software programs years in advance increases risk of failure during implementation. 
+
+Mark Hopson with special thanks to Charles Tetreault and other contributors.
```

Comment:
@heather could you swap the author and the challenge line so that authors come first? Also bold Mark Hopson and Charles Tetreault